esphome/components/pid/climate.py

```diff
@@ -36,13 +48,26 @@
             cv.Required(CONF_DEFAULT_TARGET_TEMPERATURE): cv.temperature,
             cv.Optional(CONF_COOL_OUTPUT): cv.use_id(output.FloatOutput),
             cv.Optional(CONF_HEAT_OUTPUT): cv.use_id(output.FloatOutput),
+            cv.Optional(CONF_DEADBAND_PARAMETERS, default={}): cv.Schema(
```

Comment:
```suggestion
            cv.Optional(CONF_DEADBAND_PARAMETERS): cv.Schema(
```

(removing {} here fixes the issue)

Reply:
perfect, thanks i missed that.